tests/foreman/performance/test_candlepin_concurrent_delete.py

```diff
@@ -0,0 +1,129 @@
+"""Test class for concurrent subscription deletion"""
+from datetime import date
+from robottelo.common import conf, ssh
+
+from robottelo.performance.perf_concurrent_test import ConcurrentTestCase
+
+
+class ConcurrentDeleteTestCase(ConcurrentTestCase):
+    """Concurrent Deleting subscriptions tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(ConcurrentDeleteTestCase, cls).setUpClass()
+        # parameters for concurrent activation key test
+        cls.savepoint = conf.properties[
+            'performance.test.savepoint2_enabled_repos']
+        cls.raw_file_name = conf.properties[
+            'performance.csv.raw_ak_file_name']
+        cls.stat_file_name = conf.properties[
+            'performance.csv.stat_ak_file_name']
+
+        # add date to csv files names
+        today = date.today()
+        cls.raw_file_name = '{0}-{1}'.format(
+            today.strftime('%Y%m%d'), cls.raw_file_name)
+        cls.stat_file_name = '{0}-{1}'.format(
+            today.strftime('%Y%m%d'), cls.stat_file_name)
+
+    def setUp(self):
+        super(ConcurrentTestCase, self).setUp()
+
+        # Restore database before concurrent deleting subscribed systems
+        self._restore_from_savepoint(self.savepoint)
+
+    def _get_registered_uuids(self):
+        """Get all registered systems' uuids from Postgres"""
+        self.logger.info('---------------------------------')
+        self.logger.info('Get UUID from database:')
+
+        result = ssh.command(
+            """su postgres -c 'psql -A -t -d candlepin -c """
+            """"SELECT uuid FROM cp_consumer;"'""")
+
+        self.logger.info('---------------------------------')
+        if result.return_code != 0:
+            self.logger.error('Fail to fetch uuids.')
+            return None
+        else:
+            return result.stdout
+
+    def test_delete_sequential(self):
+        """
```

Comment:
Move `@test` line to here like `"""@Test: Delete subscriptions using 1 thread"""`